src/app/features/location/address-edit/address-edit.component.ts

```diff
@@ -38,36 +38,29 @@ export class AddressEditComponent {
    * The initially pre-selected location (displayed in addition to the search field allowing to change it).
    */
   @Input() selectedLocation: GeoLocation;
-
```

Comment:
please keep those empty lines, they make the code much more readable for me and it becomes easier to grasp which comment relates to which property